src/main/java/org/primeframework/mvc/util/CookieTools.java

```diff
@@ -40,7 +41,26 @@ public final class CookieTools {
    * @return The value or null if the cookie is empty.
    * @throws Exception If the operation fails.
    */
+  @Deprecated(forRemoval = true, since = "4.22.13")
```

Comment:
`BaseManagedCookieScope` still has two uses of the deprecated method.

Reply:
Thanks. Forgot when I was combining PR code to cover this case.